sdk/identity/azure-identity/tests/test_certificate_credential.py

```diff
@@ -170,7 +170,10 @@ def test_regional_authority():
 
         assert mock_confidential_client.call_count == 1
         _, kwargs = mock_confidential_client.call_args
-        assert kwargs["azure_region"] == region.value
+        if region == RegionalAuthority.AUTO_DISCOVER_REGION:
+            assert kwargs["azure_region"]
```

Comment:
MSAL Python does not guarantee that value will remain a `True`. Better do it this way.

```suggestion
            assert kwargs["azure_region"] == msal.ConfidentialClientApplication.ATTEMPT_REGION_DISCOVERY
```